lib/advisor/web/templates/provide_advice/advice-form.html.eex

```diff
@@ -0,0 +1,19 @@
+<form action="<%= "/provide/#{@advice_id}" %>" method="post">
```

Comment:
#{@advice_id} looks different and caught my attention... does it work like <%= @advice_id %>?